components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -34,6 +34,17 @@ export default function TeamBilling() {
     const [pendingTeamPlan, setPendingTeamPlan] = useState<PendingPlan | undefined>();
     const [pollTeamSubscriptionTimeout, setPollTeamSubscriptionTimeout] = useState<NodeJS.Timeout | undefined>();
 
+    console.log(
+        "members",
+        members.length,
+        "currency",
+        currency,
+        "teamSubscription",
+        teamSubscription,
+        "pendingTeamPlan",
+        pendingTeamPlan,
+    );
+
```

Comment:
**suggestion**: Another irrelevant-line-to-comment-on: Could we adjust font color and weight following the specs?
- I find it a bit hard to parse the information in the current state. I could not find the line of code to comment directly with the class names needed. Small changes can make a difference.
- Also, what do you think of moving the sum above the plan features as in the most recent design spec, to avoid implying that the label in the bottom right corner is an action button?

| BEFORE | AFTER |
|-|-|
| <img width="526" alt="cards-before" src="https://user-images.githubusercontent.com/120486/169900474-9618d080-a460-428f-a4a0-1c5a52eb2adb.png"> | <img width="526" alt="cards-after" src="https://user-images.githubusercontent.com/120486/169900478-af2db4ed-e674-4ac9-a558-3ed8470831f3.png"> |